ti/savings/savings.go

```diff
@@ -65,6 +65,11 @@ func ParseAndUploadSavings(ctx context.Context, workspace string, log *logrus.Lo
 		}
 	}
 
+	// If the Step Fails then we should return the savings state only for restore-cache (CACHE_INTEL). This is to match the current behavior in K8
```

Comment:
Add a comment stating, no savings parsing should happen after this line of code except cache intel.

Reply:
Done